client/app/components/dynamic-table/index.js

```diff
@@ -12,8 +12,8 @@ function DynamicTable($sanitize) {
   this.orderByReverse = false;
 
   this.pageChanged = () => {
-    const first = this.count * (this.page - 1);
-    const last = this.count * (this.page);
+    const first = this.itemsPerPage * (this.page - 1);
+    const last = this.itemsPerPage * (this.page);
```

Comment:
I think it's because `this.count` [is bound](https://github.com/rockwotj/redash/blob/bc3691b5aedf5ec069690c27274953e01b083ee7/client/app/components/dynamic-table/index.js#L68) and since the parent component is not specifying a value, it's undefined. Maybe a better fix is to remove `itemsPerPage` and have this be `(this.count || 15)`?